libraries/botbuilder-core/src/botAdapter.ts

```diff
@@ -112,17 +112,6 @@ export abstract class BotAdapter {
         return this.turnError;
     }
 
-    /**
-     * Sets an error handler that can catch exceptions in the middleware or application.
-     *
-     * @remarks
-     * The error handler is called with these parameters:
-     *
-     * | Name | Type | Description |
-     * | :--- | :--- | :--- |
-     * | `context` | [TurnContext](xref:botbuilder-core.TurnContext) | The context object for the turn. |
-     * | `error` | `Error` | The Node.js error thrown. |
-     */
     public set onTurnError(value: (context: TurnContext, error: Error) => Promise<void>) {
         this.turnError = value;
     }
```

Comment:
Note: This, and other setter doc removals like it are due to this error:

```cmd
Error: src/botAdapter.ts:126:5 - (ae-setter-with-docs) The doc comment for the property "onTurnError" must appear on the getter, not the setter.
 | Error: src/testAdapter.ts:111:5 - (ae-setter-with-docs) The doc comment for the property "enableTrace" must appear on the getter, not the setter.
 | Error: src/turnContext.ts:780:5 - (ae-setter-with-docs) The doc comment for the property "locale" must appear on the getter, not the setter.
 | Error: src/turnContext.ts:798:5 - (ae-setter-with-docs) The doc comment for the property "responded" must appear on the getter, not the setter.
```

https://api-extractor.com/pages/messages/ae-setter-with-docs/

It _can_ be ignored, if we want. Thoughts?

Reply:
I think it should be ignored (or maybe set to a warning if possible). The JSDoc comments still show up in intellisense, so I think this is more of a convention than a technical requirement.